.github/workflows/phase6_graph_test.yml

```
@@ -0,0 +1,66 @@
+name: Phase 6 Graph Verification
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)